OpenRA.Mods.Common/Traits/ExternalCaptures.cs

```diff
@@ -72,20 +72,12 @@ public Order IssueOrder(Actor self, IOrderTargeter order, Target target, bool qu
 		static bool IsValidOrder(Actor self, Order order)
 		{
 			// Not targeting an actor
-			if (order.ExtraData == 0 && order.TargetActor == null)
+			if (order.Target.Type != TargetType.FrozenActor && order.Target.Type != TargetType.Actor)
```

Comment:
It could be refactored here like in Infiltrates.IsValidOrder and EngineerRepair.IsValidOrder for consistency.
```
if FrozenActor.
if Actor.. 
return false
```